schema.sql

```diff
@@ -208,6 +223,7 @@ CREATE TABLE IF NOT EXISTS `houses` (
   `highest_bidder` int(11) NOT NULL DEFAULT '0',
   `size` int(11) NOT NULL DEFAULT '0',
   `beds` int(11) NOT NULL DEFAULT '0',
+  `type` ENUM('House', 'Guildhall') NOT NULL DEFAULT 'House',
```

Comment:
full lowercase

Reply:
For guild_transactions, the enums are full uppercase. Would that be better? Enums in the source also seems to be an all upper-case situation?

Reply:
Either way works, it's just that it's easier to convert everything to lower case, we have code for that in `tools.cpp` so it's easier to ensure the value. Definitely don't mix casing.

In SQL, there's the convention that keywords are written uppercase, and values (names, data) should use lowercase to differentiate. Conversely, in C++, there's the convention that constants should use uppercase with underscores.